sys/include/net/ieee802154.h

```diff
@@ -78,12 +78,24 @@ extern "C" {
 /** @} */
 
 /**
- * @brief   Flag for @ref ieee802154_set_frame_hdr to indicate to ignore @p dst
- *          and @p dst_len and send broadcast.
- * @note    This flag is RIOT internal and shall not be used in the FCF of
- *          packets send over the air
+ * @brief   Special address defintions
+ * @{
+ */
+/**
+ * @brief   Static initializer for broadcast address
+ */
+#define IEEE802154_ADDR_BCAST               { 0xff, 0xff }
```

Comment:
Why? This group defines special IEEE 802.15.4 addresses (`IEEE802154_ADDR`). So if other special addresses are later added this would look as unified IMHO.:

```
#define IEEE802154_BCAST_ADDR   { 0xff, 0xff }
#define IEEE802154_FOO_ADDR      ....
#define IEEE802154_BAR_ADDR      ....